### PR TITLE
Feature/frontend

### LINKED ISSUE #31: Wrong Log Expand
When adding a new log and attempting to see details in log page, the wrong log is expanded
Steps to replicate:
- Create new log
- Go to logs and expand most recent log

### PR DESCRIPTION
**Pull request by Malizma333**
Fixed bug with how ids were assigned

### CODE REVIEW

**Comment by Malizma333** [comment]
Documentation needs merge before this

**Comment by cjhr95** [comment]
There are some duplicate changes here from #32

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -8,21 +8,22 @@
...
 		<h1 style="font-size: 20px; margin-top:-1ch">
-			There are two main pages you will need to navigate between, the Flight Recorder and the Flight
-			Logs. <br />
-			The Flight Recorder does exactly what the name suggests, that is where you will go to enter information
-			and record your flight.<br />
-			Once a flight has been recorded, that information will be stored within the app, where it can be
-			viewed in the Flight Logs. <br />
-			From there, you can view, manage, and export any previously recorded flight logs. <br />
+			- Compakt is compatible with both mobile and desktop platforms that have a connection to the internet. <br>
+			- The weather API used to gather data is based on the general Rolla area, not specifically our testing location <br>
```
Comment (cjhr95) [inline src/routes/+page.svelte]: Change this line to not mention Rolla, we may use it in other cities in the future. Make it more general, like:
```suggestion
			- The weather API used to gather data is based on the general area of the city provided, not specifically the testing location <br>
```

**Review by cjhr95 — CHANGES_REQUESTED** [review]

**Review by cjhr95 — APPROVED** [review]
This'll do

**Pull request merged**